Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -0,0 +1,126 @@
+#import "SentryNetworkTracker.h"
+#import "SentryOptions+Private.h"
+#import "SentryPerformanceTracker.h"
+#import "SentrySDK+Private.h"
+#import <objc/runtime.h>
+
+static NSString *const SENTRY_NETWORK_REQUEST_TRACKER_SPAN_ID
+    = @"SENTRY_NETWORK_REQUEST_TRACKER_SPAN_ID";
+
+@interface
+SentryNetworkTracker ()
+
+@property (nonatomic, strong) SentryPerformanceTracker *tracker;
+
+@end
+
+@implementation SentryNetworkTracker
```

Comment:
`h`: Does this class work with all different types of tasks?

- `NSURLSession’s uploadTaskWithRequest:fromData:`
-  `NSURLSession’s downloadTaskWithURL:`
- `NSURLSession’s streamTaskWithHostName:port: or streamTaskWithNetService:`

Reply:
For now, let's handle just DataTask, DownloadTask and UploadTask.
Since stream is usually used to stay connected for long doesn't seeing interesting to monitor its performance.